apps/web/src/pages/templates/components/StepEditorSidebar.tsx

```diff
@@ -62,6 +72,7 @@ export const StepEditorSidebar = ({ children }: { children: ReactNode }) => {
         navigate(basePath);
       }}
       data-test-id="step-editor-sidebar"
+      onSubmit={handleSubmit(onSubmitHandler, onInvalid)}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** What happened here -- did someone make changes that caused this or is this related to our feature work?‏

Reply:
Yes, the Sidebar component was refactored and the form component inside the Sidebar is conditionally rendered now only when onSubmit is passed. Previously, the form used to render always.

Reply:
Ah sorry this is my fault! But also points out the pattern of components that do too much and implicitly have extra behavior